dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -83,6 +84,17 @@ class R2_4(enum.Enum):
     TORCH_REPO_BRANCH = "-b v2.4.0-rc8"
     TORCH_XLA_REPO_BRANCH = "-b v2.4.0-rc8"
 
+  class R2_5(enum.Enum):
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-linux_x86_64.whl"
```

Comment:
hi, @zpcore, I saw we would add CUDA support wheel for 12.1 and 12.4 for 2.5, do we want to keep test with 12.1 like 2.4 too?

Reply:
It’s fine for now. This is for the final release performance test. We can add it near the end of the release.

Reply:
Thanks, does it matter if add it early? since we included this in this PR, and we might plan to merge this PR soon to enable nightly tests, so wanna confirm before merge

Reply:
Yes, we can add it with the RC version whl link if we have. We can enable it here https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/4ceb5e5fd528d61c86d321fcb935db22a46192ef/dags/pytorch_xla/pytorchxla-torchbench-release.py#L35 once the wheel is ready.